docs/guide/blog.md

```diff
@@ -84,18 +108,39 @@ func NewHandler(k keeper.Keeper) sdk.Handler {
 }
 ```
 
-`case *types.MsgCreatePost` handles messages of type `MsgCreatePost`, calls `CreatePost` method and returns back the response.
+The `case *types.MsgCreatePost` statement handles messages of type `MsgCreatePost`, calls the `CreatePost` method, and returns back the response.
 
 Every module has a handler function like this to process messages and call keeper methods.
 
-### Process Messages
+## Register the Query Handler 
```

Comment:
@soniasingla I wasn't sure where to add the step to register the query handler, so I guessed here
added these 2 steps line 119 and 121-128

Reply:
```suggestion
## Add GRPC to the Module Handler
```

Reply:
@barriebyron we need to move this step before the section `## Write Data to the Store`